packages/core/src/browser/frontend-application.ts

```diff
@@ -9,16 +9,26 @@ import { inject, injectable, named } from 'inversify';
 import { ContributionProvider, CommandRegistry, KeybindingRegistry, MenuModelRegistry } from '../common';
 import { ApplicationShell } from './shell';
 import { Widget } from "./widgets";
+import { ILogger } from '../common';
 
 /**
  * Clients can implement to get a callback for contributing widgets to a shell on start.
  */
 export const FrontendApplicationContribution = Symbol("FrontendApplicationContribution");
 export interface FrontendApplicationContribution {
+
+    /**
+     * Called when the application is started.
+     */
+    onStart?(app: FrontendApplication): void;
+
+
```

Comment:
empty line

Reply:
https://palantir.github.io/tslint/rules/no-consecutive-blank-lines/ can catch such cases

Reply:
Nice I'll add that thx

Reply:
Would also be good if our autoformatting would clean that already.

Reply:
Yes let's fix that as part of #413